### ISSUE TITLE
Add DateAnonymizer 

### ISSUE THREAD

**Issue by SimonMellerin** [makinacorpus/DbToolsBundle#32]

**Comment by SimonMellerin** [comment]
Also add options to be able to anonymize int by adding noise.

```php
    #[ORM\Column]
    // This will add to the value a random interval between -P1D and P1D
    #[Anonymize(type: 'date', options: ['delta' => 'P1D'])]
    private ?int $age = null;
```

**Comment by SimonMellerin** [comment]
Won't be easy to do without https://github.com/makinacorpus/php-query-builder/issues/5

**Comment by SimonMellerin** [comment]
This has been implemented, see #127

It is already on the main branch and will be available in the next release (1.1.x expected for the next week)
